design/gateway-apis-implementation.md

```diff
@@ -62,69 +61,77 @@ Project Contour (the organisation) may investigate a Gateway API based layer 4 s
 
 ### Contour Gateway model
 
-Contour considers a Gateway to describe a single Envoy deployment, deployed and managed by something outside of itself.
-Contour expects to be supplied with the full name (name and namespace) of a single Gateway, which it will watch and update the status of.
-This will be a configuration file parameter.
-When a Gateway is not supplied, Contour will not do Gateway API processing, or in other words, the existing `--experimental-service-apis` flag will be obsoleted and removed).
-Inside Contour will merge Listeners within its Gateway. (See the detailed design for the exact rules Contour will use for this.)
+Contour considers a Gateway to describe a single Envoy deployment.
+Contour expects to be supplied with a `ControllerName` which is required to match a `GatewayClass`.
```

Comment:
Contour's expectation is that there's only one `GatewayClass` for a given `ControllerName`, and only one `Gateway` for that `GatewayClass`, right? Might be helpful to state that explicitly here since that's different than what the API itself allows.

Reply:
Agreed, we should say that Contour will only use a single Gateway and GatewayClass, and say that the rules for choosing a winner in the event there's more than one are specified in this document.

Reply:
👍